src/build/optimize-streams.ts

```diff
@@ -60,7 +60,7 @@ export class GenericOptimizeStream extends Transform {
       } catch (error) {
         logger.warn(
             `Unable to optimize ${this.validExtension} file ${file.path}`,
-            {err: error});
+            {err: error.message || error});
```

Comment:
I see. Ideally in that sort of case we'd have also already emitted a nice Warning from the analyzer too.